plugins/labhub.py

```diff
@@ -225,9 +231,11 @@ def mark_cmd(self, msg, match):
                 return ('The pull request {mr_link} is marked *work in progress'
                         '*. Use `{bot_prefix} mark pending` or push to your '
                         'branch if feedback from the community is needed '
-                        'again.'.format(
+                        'again.\n@{user_login}, please check your pull '
+                        'request.'.format(
                             mr_link=mr.url,
-                            bot_prefix=self.bot_config.BOT_PREFIX)
+                            bot_prefix=self.bot_config.BOT_PREFIX,
+                            user_login=login)
```

Comment:
if `login` is `''` , the markdown will be `@,` , which would be a bug.  So a small fix is needed here.

Otherwise this looks great.  Could you squash it into a single commit when you do your next push.

Reply:
What would you suggest doing here? Should I just omit the ping if the login can't be found?

Reply:
That is probably the best approach.

Reply:
And so you dont think I am crazy ..
There are lots of reasons why the username may not exist.  I suspect that this isnt very likely to occur when an MR is first created, but we have people vanish after they create their MR.  GitHub now has a @ghost account which it substitutes in, but I dont know what GitLab and other services respond to these odd scenarios (corobo will also support BitBucket and other services soon also).